app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -365,4 +369,14 @@ protected void onDestroy() {
         BOOKMARK,
         MORE
     }
+
+    /**
+     * Load default language in onCreate from SharedPreferences
+     */
+    private void loadLocale(){
+        final SharedPreferences preferences = getSharedPreferences("Settings", Activity.MODE_PRIVATE);
+        final String lang = preferences.getString("language","");
```

Comment:
Space after `,`

Reply:
If possible please use the full word `language` as the variable name. In Java we prefer to use full words.
(acronyms such as "UI" or "XML" are OK though)